doc/skrub_pipeline.rst

```diff
@@ -395,3 +395,72 @@ Finally, there are other situations where using :func:`deferred` can be helpful:
   expressions on dataframes.
 - See :ref:`example_tuning_pipelines` for an example of hyper-parameter tuning using
   skrub pipelines.
+
+Subsampling
+~~~~~~~~~~~
```

Comment:
Not a request for change or action, just a comment: Super improved rewrite (@jeromedockes close this when you see it).

Reply:
thanks! I was just now working on further reducing the user guide part and adding an actual example in the gallery with the employee salaries, but do you think we should keep it the way it is instead? not sure if there is much more to say in an actual example (still slightly unsure about the respective roles of the user guide and the examples)